### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by SergeyPirogov** [bayandin/devtools-proxy#91]
On my current project I need to run devtools-proxy in Windows 10. As I understand now it's impossible to do this. Am I right? If yes, do I need to write dev-tools.bat the same manner as .sh or it requires additional changed in library?

**Comment by bayandin** [comment]
Yes, you're right. The Proxy itself ([proxy.py](https://github.com/bayandin/devtools-proxy/blob/master/devtools/proxy.py)) is fully compatible with Windows already. The thing that is incompatible with Windows is a Chrome wrapper ([chrome-wrapper.sh](https://github.com/bayandin/devtools-proxy/blob/master/devtools/chrome-wrapper.sh)). I see 2 possible solutions:
- write `chrome-wrapper.bat` (or `.ps1`) which will do the same as `chrome-wrapper.sh` does.
- write `chrome-wrapper.py` in Python and replace `chrome-wrapper.sh` with it. Then there is no need to support 2 different wrappers for different systems (I prefer this way, to be honest, but it will require more time).

**Comment by SergeyPirogov** [comment]
Will try to run it on Linux and understand how it works, then will try to create bat first. If I am not mistaken, by using chrome-wrapper as binary in ChromeOptions, we still should set the path to chrome driver binary as well?

**Comment by bayandin** [comment]
> we still should set the path to chrome driver binary as well?

Did you mean Chrome (not chrome driver)? If so, then it's not required, In the wrapper, I've tried to [mimic](https://github.com/bayandin/devtools-proxy/blob/6386e32312b6162b1b4f969cec92cdf8976f84d6/devtools/chrome-wrapper.sh#L10-L19) chromedriver's [heuristic](https://chromium.googlesource.com/chromium/src/+/2729e442b1172c5094503a03fe356c8580bb919d/chrome/test/chromedriver/chrome/chrome_finder.cc) for finding Chrome's binary. But if you want to use custom binary (which is not in default locations), you can add an argument [`--chrome-binary`](https://github.com/bayandin/devtools-proxy/blob/6386e32312b6162b1b4f969cec92cdf8976f84d6/devtools/chrome-wrapper.sh#L52) to ChromeOptions' args.

**Comment by SergeyPirogov** [comment]
I mean [chrome driver](https://sites.google.com/a/chromium.org/chromedriver/downloads)

**Comment by bayandin** [comment]
devtools-proxy doesn't require any changes of chromedriver usage (it requires only additional parameters in ChromeOptions). So if you set it without proxy, then you need to set it with the proxy.
